docs/workflow/development.md

```diff
@@ -1,84 +1,113 @@
 
-**Author:** The person that created this card (issue/pull request). Responsible for answering questions (issues/pull requests) and implement/discuss requested changes (pull requests).
+| _Open_   | _In Progress_            | _Request for Review_    | _Done_   |
+| -------- | ------------------------ | ----------------------- | -------- |
+| -------- | Developer is responsible | Reviewer is responsible | -------- |
+
+??? question "The _hidden_ column"
+
+    The _hidden_ column is a special column that is not visible on the project board.
+    It contains all open issues, that are not in _Open_ and that currently should not be worked on.
+    This is to keep the project board clean and focused on the current iteration.
 
 #### Open
 
-The *Open* section contains by the dev-leads selected **issues** that are important and of high priority for the current iteration. However, if you want to work on issues that are not in *Open*: Feel free to do so.
+The _Open_ section contains by the dev-leads selected **issues** that are important and of high priority for the current iteration.
 
-**Move** (or **add**) an issue into *In Progress* **and** assign yourself when you started working on it.
+!!! note
 
-#### In Progress
+    If you want to work on issues that are not in _Open_:
+    Feel free to do so, but please create and issue beforehand and assign yourself to it, so that others know that you are working on it.
 
-The *In Progress* section contains:
+Move (or add) an issue into _In Progress_ **and** assign yourself when you started working on it.
 
-- **issues** that have an assignee, but no open pull request. As soon as there is an open pull request fixing this issue, the issue card should be replaced by this pull request card (removing the Issue from the Project but not closing it as it is not fixed in the main yet). The issue must then be mentioned with the `fixes #Issue_NO` in the pull requests description.
-- **pull requests** that are not ready for review yet and are currently wip.
+#### In Progress
 
-**Note:** `fixes #Issue_NO` is a keyword on github. Github will automatically close the mentioned issue whenever the corresponding pull request was merged. Do **not** close issues that have not being fixed in the main yet (even if there is a pull request for it)!
+The _In Progress_ section contains:
 
-**Move** a pull request into *Request for Review* when you have finished your work **and** tested the pull request yourself on relevant platforms. At this stage a pull requests' description should be finalized (fill out the template properly).
+-   **issues** that have an assignee, but no open pull request.
+    As soon as there is an open pull request fixing this issue, the issue card should be replaced by this pull request card (removing the Issue from the Project but not closing it as it is not fixed in the main yet).
+    The issue must then be mentioned with the `fixes #Issue_NO` in the pull requests description.
```

Comment:
```suggestion
    The issue must then be mentioned with the [`fixes` keyword](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue) in the pull requests description.
```
Then we can remove the note below.
As much as I like admonitions, I think this page is quite overladen with with them already.

Reply:
Okay :(